src/app/pages/quickMaps/components/sideNavContent/sideNavContent.component.scss

```diff
@@ -148,9 +150,25 @@ h3 {
     font-size: 16px;
     color: $color_blue_form;
   }
+
   &.mat-button-disabled {
     & i {
       color: $color_grey;
     }
   }
+}
+
+::ng-deep {
```

Comment:
Don't use ::ng-deep at root or it will affect everything, everywhere.
Use :host ::ng-deep to at least keep it within the component.

Also the sub-selectors need "&" on the start :-)